config/kubernetes-incubator/org.yaml

```diff
@@ -147,6 +147,7 @@ members:
 - sarahnovotny
 - sbezverk
 - sebgoa
+- serathius
```

Comment:
Hey there @serathius, a couple things:
- Org membership changes should be their own separate commit and have the commit message be something like: `Add serathius to kubernetes-incubator org`. 
- Is joining incubator intended to help with migration to @kubernetes-sigs?  With [incubator's deprecation](https://github.com/kubernetes/community/issues/1922#issuecomment-516974448) we're not adding new members to @kubernetes-incubator unless it's absolutely essential. cc @brancz

Reply:
Thanks for review, I created separate PR for kubernetes-incubator https://github.com/kubernetes/org/pull/1379